encord/objects/coordinates.py

```diff
@@ -272,7 +273,7 @@ def to_dict(self) -> dict:
         return {str(idx): {"x": value.x, "y": value.y} for idx, value in enumerate(self.values)}
 
 
-class Visibility(Flag):
```

Comment:
Check where else this is used and whether this change doesn't break other stuff. Ideally we want to modify as few things as possible